.circleci/config.yml

```diff
@@ -3,49 +3,81 @@
 # Check https://circleci.com/docs/2.0/language-java/ for more details
 #
 version: 2
-jobs:
-  build:
+references:
+  container_jdk8py37: &container_jdk8py37
```

Comment:
I know the name is just a name, but I'm just wondering - is it specifying python 3.7 in any way?

Reply:
The docker cumuluss/openjdk:8-py37 has python 3.7 installed.